airbyte-webapp/.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+enable-pre-post-scripts=true
```

Comment:
:information_source: Required for `pnpm` to execute the `prestart` etc scripts. It doesn't do so by default, but we're using the heavily for Orval compilation.